packages/ui-kit/src/Textarea/index.tsx

```diff
@@ -113,14 +115,25 @@ function TextareaComponent(
     [onChange],
   );
 
-  const getPlaceholder = () => {
+  const getPlaceholders = () => {
     if (!placeholder) return;
```

Comment:
If only `showTopPlaceholder` and `topPlaceholder` props are set, nothing will be shown. Maybe it's a good idea to modify conditions in this function to allow any combination of props?